lib/yukari.js

```diff
@@ -525,6 +525,33 @@ Yukari.prototype.update = function(callback) {
 };
 
 /**
+ * update  by json
+ * @data jsonData
+ */
```

Comment:
/**
 * update by JSON
 * @param {Object} jsonData the json data to be updated
 * @param {Function} callback the callback function
 */